databases/week2/homework.sql

```diff
@@ -0,0 +1,85 @@
+-- Active: 1660903376490@@127.0.0.1@3306@hyf_lesson_1
+
+--Part 1: Working with tasks
+
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+    task (
+        title,
+        description,
+        created,
+        updated,
+        due_date,
+        status_id,
+        user_id
+    )
+VALUES
+(
+        "HackYourFuture",
+        "to be a fullstack webdeveloper",
+        "2022-04-22",
+        "2022-08-21",
+        "2022-12-20",
+        2,
+        1
+    );
```

Comment:
Why this much line for this, pls keep your lines of code as few as possible.
this could have been done in 2 lines as below
```
INSERT INTO task (title, description, created, updated, due_date, status_id, user_id) VALUES
('Homework', 'Database homework for week2','2021-04-01','2022-08-25','2022-08-25',1,1);
```

Reply:
this is automatically formatted but will change it